### PR TITLE
styling for course collections and lists

### PR DESCRIPTION
**Pull request by abeglova**
#### Pre-Flight checklist

- [ ] Screenshots and design review for any changes that affect layout or styling
  - [ ] Desktop screenshots
  - [ ] Mobile width screenshots
- [ ] Testing
  - [ ] Code is tested
  - [ ] Changes have been manually tested

Course collection desktop:
Screen Shot 2022-03-29 at 11 09 45 PM
Screen Shot 2022-03-29 at 11 09 56 PM
Screen Shot 2022-03-30 at 9 09 22 AM

Course collection mobile:
Screen Shot 2022-03-29 at 11 10 08 PM
Screen Shot 2022-03-29 at 11 10 21 PM

Course list desktop:
Screen Shot 2022-03-29 at 11 10 36 PM
Course list mobile:
Screen Shot 2022-03-29 at 11 10 53 PM

#### What are the relevant tickets?
None

#### What's this PR do?
This pr improves the styling for course collections and lists

#### How should this be manually tested?
Pull down the latest ocw-www repo.
Set WWW_CONTENT_PATH=<your path>/ocw-www 
Run npm run start:www

View a course collection. One is http://localhost:3000/collections/energy/

Course collections should look like the following design:

https://invis.io/DN12GXJ28MXA#/464971999_Course_Collection_B

There isn't a mobile design or a design for the course list page but both of those should look good.

The cover image doesn't load locally but will work on RC.

### CODE REVIEW

**Comment by Ferdi** [comment]
* Can we have a screenshot of a course collection page with more than one course list , each of them with a description
* We need to keep the level in the card  (graduate / undergraduate).  Not sure how that was removed from the design.

Screen Shot 2022-03-30 at 9 06 18 AM

**Comment by abeglova** [comment]
Updated screenshots
Screen Shot 2022-03-30 at 10 29 17 AM
Screen Shot 2022-03-30 at 10 29 44 AM

File: www/assets/css/course-collection.scss [hunk 1]
```diff
@@ -25,18 +25,70 @@
   }
 }
 
+h1.course-list-title {
+  color: $highlight-red;
+  text-transform: none;
+  font-size: $heading-two-font-lg;
```
Comment (MAbdurrehman12) [inline www/assets/css/course-collection.scss:31]: I think we can remove this font-size as `$font-xl` is already being applied below or vice versa.

**Review by MAbdurrehman12 — APPROVED** [review]
Looks good! 
Just a small nit

File: www/assets/css/course-collection.scss (hunk 1 above)
Comment (abeglova) [reply www/assets/css/course-collection.scss:31]: So $heading-two-font-lg is 28 px
But $font-xl is 24px
I needed the top title to be bigger than the list titles

Comment (abeglova) [reply www/assets/css/course-collection.scss:31]: Oh wait there's 2 of the same font size in the same class
Fixing!

Comment (MAbdurrehman12) [reply www/assets/css/course-collection.scss]: Other than this, I think we are good!
If @Ferdi is okay with the design

Comment (Ferdi) [reply www/assets/css/course-collection.scss]: Let's merge it . thank you both !

**Pull request merged**